xchangelog

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+# xchangelog PKGNAME - open package changelog
+
+XBPS_DISTDIR=$(xdistdir) || exit 1
+
+pkg=$1
+[ -z "$pkg" ] && pkg=$(basename "$PWD")
+
+if [ -f "$pkg" ]; then
+	template="$pkg"
+elif [ -f "$pkg/template" ]; then
+	template="$pkg/template"
+elif [ -f "$XBPS_DISTDIR/srcpkgs/$pkg/template" ]; then
+	template="$XBPS_DISTDIR/srcpkgs/$pkg/template"
+else
+	echo "No such template: $pkg"
+	exit 1
+fi
+
+case $template in
+	/*) ;;
+	*) template=./$template ;;
+esac
+
+. "$template"
+
+if [ -z "$changelog" ]; then
+	echo "No changelog defined in template"
+	exit 1
+fi
+
+if curl -sI -- "$changelog" | grep -i "^content-type: text/plain"; then
+	curl -s -- "$changelog" | "${PAGER:-less}"
```

Comment:
maybe fall back to `cat` if `less` isn't available

Reply:
Or `more`?

Reply:
personally i'd not want to subject someone to more, but idk

Reply:
Honestly, if somebody is running a system without `less` installed, I don't know what else they'd expect.

Reply:
I went ahead and made outputting to stdout the fallback. I think a case could be made for using `more` instead, but I don't know.